regression/cbmc/pointer-to-struct-with-flexible-array-member-as-parameter-to-entry-point/main.c

```diff
@@ -0,0 +1,15 @@
+#include <assert.h>
+#include <stdlib.h>
+
+typedef struct ST ST;
+typedef struct ST
+{
+  // int id;
+  // ST *c[];
+  int c[];
+} ST;
```

Comment:
I tried to compile this with gcc and it turns out structs with only a flexible array member are illegal. So this would need to contain `int id; int c[];`.

Reply:
Of course! Thanks for checking ... I keep forgetting :sweat_smile: